apps/api/tests/test_guest.py

```diff
@@ -124,5 +122,3 @@ def test_invalid_guest_verification_is_unauthorized(
     )
 
     assert res.status_code == 401
-
-    mock_remove_guest_key.assert_awaited_once_with("edu.caltech.beaver")
```

Comment:
Chore: We can use [`AsyncMock.assert_not_awaited`](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.AsyncMock.assert_not_awaited) to check that the guest key is not removed.